examples/advection/tests/adv_1D_1D_k3.hyperrectangle_03.json

```diff
@@ -32,6 +32,9 @@
         "PerformanceLogAllCalls": "false",
         "DiagnosticsWarmUpIterations": "0"
     },
+    "AdvectionOperation": {
+        "SkewFactor": "0.0"
+    },
```

Comment:
I had to add these lines. Because else I get segmentation fault on Travis CI!? This is definitely a bug. However, let's merge and tackle this issue later!